packages/gen-ai/bff/openapi/src/gen-ai.yaml

```
@@ -922,27 +972,48 @@
             ]
-                    
-            from llama_stack_client import LlamaStackClient
-                    
-            client = LlamaStackClient(base_url=LLAMA_STACK_URL)
-                    
+
+            for file_info in files_to_upload:
+                with open(os.path.join(FILES_BASE_PATH, file_info["file_name"]), 'rb') as file:
+                    uploaded_file = client.files.create(file=file, purpose=file_info["purpose"])
+                    client.vector_stores.files.create(
+                        vector_store_id=vector_store.id, 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)